csrankings-f.csv

```diff
@@ -291,6 +291,7 @@ Feza Buzluca,Istanbul Technical University,http://web.itu.edu.tr/buzluca,nQqXUsk
 Fidel Cacheda,University of A Coruña,https://pdi.udc.es/es/File/Pdi/H399E,NOSCHOLARPAGE
 Fikret Gürgen,Boğaziçi University,https://www.cmpe.boun.edu.tr/~gurgen,B8EWl8gAAAAJ
 Fikret S. Gürgen,Boğaziçi University,https://www.cmpe.boun.edu.tr/~gurgen,B8EWl8gAAAAJ
+Filip Ilievski,University of Southern California,https://usc-isi-i2.github.io/ilievski,4ZScBc0AAAAJ

```

Comment:
Appears to be a research scientist (non-TT), and thus not eligible for inclusion. Please clarify or remove.